src/main/java/nl/goodbytes/xmpp/xep0363/Component.java

```diff
@@ -114,6 +118,29 @@ protected IQ handleIQGet( IQ iq ) throws Exception
     {
         final Element request = iq.getChildElement();
         final Collection<String> namespaces = Arrays.asList( NAMESPACE, NAMESPACE_EXP );
+        // Implements the TYPE_IQ jabber:iq:version protocol (version info xep-0092). Allows
+        // XMPP entities to query each other's application versions.  The server
+        // will respond with its current version info.
+        if ("query".equals(request.getQName().getName()) && "jabber:iq:version".equals( request.getNamespaceURI())) {
+            try {
+                Element answerElement = DocumentHelper.createElement(QName.get("query", "jabber:iq:version"));
+                answerElement.addElement("name").setText("Http File Upload Component");
+                answerElement.addElement("version").setText("1.2.0");
```

Comment:
For bonus points, try to make this use the version number that is defined in the Maven pom.xml file. If we don't do that, then I will likely forget to update this code, every time we do a new release!